install/streams.csv

```diff
@@ -2013,10 +2013,10 @@ TLED,20,200,true,false,true,2010-09-05T05:00:02Z,2012-02-21T01:00:00Z
 TLZ,10,1,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,10,100,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,20,1,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
-TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,21,50,false,false,false,2020-01-15T02:16:00Z,2020-06-16T00:00:00Z
-TLZ,21,200,false,false,true,2020-01-15T02:16:00Z,2020-07-02T02:31:00Z
+TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
+TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
```

Comment:
If we want it very specific, end date looks like around `2016-02-12T01:25:00`
And data for LC 21 starts at `01:25:12` based on the data plot on `2016.043`

Reply:
yeah the delta entry was  rounded apparently

Reply:
will clean that up :)